src/main/_docs/assemblies/assemblies-assembly-lifecycle.md

```diff
@@ -7,7 +7,7 @@ permalink: /:categories/assembly-lifecycle/
 ---
 {% include base.html %}
 
-You can create new Che products for re-distribution by generating a custom assembly. An assembly is a binary packaging of IDE extensions (written in Java / JavaScript), Che extensions (deployed in the Che server), workspace extensions (deployed in a workspace agent), custom agents, stacks, templates, custom CLI, and brand elements. When these are packaged together, a complete binary that can be run identically to how you run the standard Che assembly is provided. 
+You can create new Che products for re-distribution by generating a custom assembly. An assembly is a binary packaging of IDE extensions (written in Java / JavaScript), Che extensions (deployed in the Che server), workspace extensions (deployed in a workspace agent), custom agents, stacks, templates, custom CLI, and brand elements. When these are packaged together, a complete binary that can be run identically to how you run the standard Che assembly is provided.

```

Comment:
I see that you have removed some spaces. In general those are in place because they allow to properly format the content, once it is generated with Jekyll. 
Have you checked if those removals are changing the formatting?